wandb/sdk/internal/tb_watcher.py

```diff
@@ -462,6 +463,10 @@ def _flush(self) -> None:
             )
             print("\t" + ("\n\t".join(dropped_keys)))
         self._data["_step"] = self._step
+        self._data["_timestamp"] = int(time.time())
+        self._data["_runtime"] = int(
+            self._data.get("_timestamp", time.time()) - self.start_time
+        )
```

Comment:
This isn't going to work.  We should be able to get the original timestamp from the summaries in the tfevent file.  Users can sync runs that ran in the past.  We should already be adding "_timestamp" from the original tfevent file [here](https://github.com/wandb/client/blob/master/wandb/integration/tensorboard/log.py#L76).  Instead of settings `self.start_time` to `time.time()` we should default it to None and set it to the first timestamp we get from the tfevent file.